site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -315,7 +315,7 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `agones.extensions.mutatingWebhook.disableCaBundle`      | Disable ca-bundle so it can be injected by cert-manager.  If not set, this parameter defaults to the value of `.Values.agones.controller.mutatingWebhook.disableCaBundle`.                                                                                                                                                                                                                                                          | `false`                                   |
 | `agones.extensions.allocationBatchWaitTime`              | Wait time between each allocation batch when performing allocations in controller mode                                                                                                                                                                                                                           | `500ms`                                   |
 | `agones.extensions.pdb.minAvailable`                     | Description of the number of pods from that set that must still be available after the eviction, even in the absence of the evicted pod. Can be either an absolute number or a percentage. Mutually Exclusive with maxUnavailable                                                                                | `1`                                       |
-| `agones.extensions.pdb.maxUnavailable`                   | Description of the number of pods from that set that can be unavailable after the eviction. It can be either an absolute number or a percentage Mutually Exclusive with `minAvailable`                                                                                                                           | \`\`                                      |
+| `agones.extensions.pdb.maxUnavailable`                   | Description of the number of pods from that set that can be unavailable after the eviction. It can be either an absolute number or a percentage Mutually Exclusive with `minAvailable`. If not set, this parameter defaults to the value of `.Values.agones.controller.pdb.maxUnavailable`.                                                                                                                           | \`\`                                      |
```

Comment:
This doesn't feel right. The `minAvailable` defaults to something sane here, and `maxUnavailable` is exclusively with `maxUnavailable`. I think it's safer not to default this one.

_That said_, and this is not your fault, can you fix a typo on this line? Add a period between "percentage" and "Mutually". It should read:

`It can be either an absolute number or a percentage. Mutually [...]`

Reply:
I have reverted the changes related to maxUnavailable and also corrected the missing period👍🏻